blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -220,6 +245,17 @@ Nav.propTypes = {
       group: 'Configure content',
     }),
     signInOrder: PropTypes.number,
+    logoAlignment: PropTypes.oneOf([
+      'center', 'left',
+    ]).tag({
+      label: 'Logo alignment',
+      group: 'Logo',
+      defaultValue: 'center',
+    }),
+    horizontalLinksHierarchy: PropTypes.string.tag({
```

Comment:
are you handling the default value for this for preexisting pages?

Reply:
For preexisting pages the horizontal links won't be shown - this custom field is used to populate the links but also acts as the on/off switch to display the horizontal links.  so no horizontalLinksHierarchy => no horizontal links shown.  The default is null hierarchy and thus no horizontal links shown which is what the preexisting components will do.

Reply:
oh ok, we're relying on it being null and evaluating as falsey. If it was up to me, I'd always want a default value just so  we can be explicit about this (it's bitten me before).